### PR TITLE
Add function for updateTM && review saveTM

### PR DESCRIPTION
**Pull request by AnyaTyu**
#9

### CODE REVIEW

File: common.blocks/editor/editor.js [hunk 1]
```diff
@@ -1,42 +1,32 @@
 modules.define('editor', ['i-bem__dom', 'jquery'], function(provide, BEMDOM, $) {
 
 provide(BEMDOM.decl(this.name, {
-    _onFocusIn: function(e) {
+    onFocusIn: function(e) {
         this._target = $(e.target).parents('.editor__unit');
```
Comment (tadatuta) [inline common.blocks/editor/editor.js:5]: this.elem('unit')

File: common.blocks/editor/editor.js [hunk 2]
```diff
@@ -1,42 +1,32 @@
...
 
-        return data;
+        window.segments[index].status = elem.getMod('checked');
+        console.log('segment:', window.segments[index]);
     }
 }, {
     live: function() {
-        this.liveBindTo('target', 'focusin', this.prototype._onFocusIn);
-        this.liveBindTo('target', 'focusout', this.prototype._onFocusOut);
+        this.liveBindTo('target', 'focusin', this.prototype.onFocusIn);
```
Comment (tadatuta) [inline common.blocks/editor/editor.js]: use chaining

File: common.blocks/editor/editor.js (hunk 1 above)
Comment (AnyaTyu) [inline common.blocks/editor/editor.js:5]: @tadatuta return array of units. I need one elem =(

Comment (tadatuta) [inline common.blocks/editor/editor.js:5]:
``` js
this.buildSelector('unit')
```

**Pull request merged**